modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeFooter.tsx

```diff
@@ -118,7 +128,19 @@ export default function ObjectDefinitionNodeFooter({
 					className="lfr-objects__model-builder-node-show-all-fields-button"
 					displayType="unstyled"
 					onClick={() => {
-						setShowAllObjectFields(!showAllObjectFields);
+						handleSelectObjectDefinitionNode();
```

Comment:
Hi Arthur, since the selectObjectDefinitionNode was never triggered by the showAllFields button. We decided not to apply related changes to fix this behavior. So, could you remove this function call, since the expected result of adding this function was not achieved? Thanks

Reply:
done!